LFS171x/indy-material/nodejs/AgentDemoScript.md

```diff
@@ -0,0 +1,319 @@
+
+But wait. There's a problem with that. What if they lied to us?  Told us they were Alice, when really they were her malicious enemy, Mallory? What we want is a trusted way to get the name of the other party. This is where Verifiable Credentials come in.
+
+As soon as the connection is established, both Alice and Faber send each other a Proof Request that asks for the name of the other party using the "Government ID" credential both are already holding. Since that Credential is owned by its Holder, the Agent software prompts the user to confirm they want to share the requested information.
+
+Alice checks her messages and sees that she has a Proof Request from Faber requesting her name. 
+
+|![NamePRNotes](images/NamePRNotes.png "Accept Proof Request Popup")|
+|:--:|
+|*Licensed under [CC By 4.0](https://creativecommons.org/licenses/by/4.0/)*|
+
+1.  Click the **Messages** tab then click **Accept**.
+
+Meanwhile, at Faber College, they too have received a "Name" Proof Request, this one from Alice, which they too accept, and resulting in Alice getting the College's official name "Faber" - per the Government ID. You'll need to refresh the browser tabs to see the latest data.
+
+
+### Step 4: It's All About Your Credentials
+
+Alice will need her transcript to apply for the job at Acme. Since Alice and Faber now have a relationship, she can go ahead and ask for her transcripts. We don't show how that happens in this demo, but let's pretend she has done that.
+
+
+> **Note: _In the Indy "Credential Issuance" protocol, there are messages for Offer (Issuer to Holder), Request (Holder to Issuer) and Issue (Issuer to Holder). The latter two must be done to enable the creation of a Verifiable Credential. While Alice could have triggered the process (and it would have made more sense!), it was easier to build this demo by having Faber start the process with the Offer step._**
+
+Faber, knowing Alice wants her transcript, starts the process. It's time for John Academic to get back to work. Go to the Faber browser tab and click on the **Issuing** menu item.
+
+
+
+1.  In the Send Credential Offer window, select a **Relationship** (Alice) then a **Credential Definition** (MyTranscript) and click **Submit**.
+
+|![SendCredOffer](images/CredOffer.png "Faber sends Cred Offer to Alice")|
+|:--:|
+|*Licensed under [CC By 4.0](https://creativecommons.org/licenses/by/4.0/)*|
+
+Going back to the Alice browser tab (refresh the browser tab), Alice sees that in her **Messages**, she has a new Credential Offer: her transcript from Faber (straight A's btw), which she accepts. 
+
+|![CredOfferTran](images/CredOfferTran.png "Accept Credential Offer Popup")|
+|:--:|
+|*Licensed under [CC By 4.0](https://creativecommons.org/licenses/by/4.0/)*|
+
+
+
+1.  Click **Credentials**.

```

Comment:
Should this be a 2? It feels like it since it, but you may have to step 1's because one is done as Faber and one is done as Alice.

Reply:
Fixed